extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +262,42 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @private
+   */
+  handleRequestsFrameUrl_() {
+    if (!this.config_['requestsFrameUrl']) {
```

Comment:
You are correct about the merging. As far as the "ONLY" part, to be clear I'm not saying at this point "let's push this PR as-is", I'm treating it more as a working PR. That is absolutely on the slate to do.